src/main/java/io/appform/dropwizard/sharding/DBShardingBundleBase.java

```diff
@@ -184,31 +188,36 @@ protected ShardBlacklistingStore getBlacklistingStore() {
         return new InMemoryLocalShardBlacklistingStore();
     }
 
+    protected ShardingBundleOptions getShardingOptions() {
+        return new ShardingBundleOptions();
```

Comment:
This should come from config. If not present default it to new object. Can take config as param to this method

Reply:
done & added default object